### ISSUE TITLE
Typescript error in PolygonGraphics

### ISSUE THREAD

**Issue by golyalpha** [AnalyticalGraphicsInc/cesium#6831]
The outlineColor property of the options object is incorrectly typed.
If I provide the property with a ColorMaterialProperty object, TypeScript is silent, but the outline is not rendered, where as, if I provide it with a Color object, TypeScript screams about missing property isConstant in the object, but Cesium renders the outline successfully.

```js
new PolygonGraphics({
    hierarchy: new PolygonHierarchy(measure.coordinates.map(coords => (
        Cartesian3.fromDegrees(coords[0], coords[1])
    ))),
    height: 0,
    material: fillMat,
    outline: true,
    outlineColor: Color.RED
})
```
![image](https://user-images.githubusercontent.com/8203992/43125738-2c328d6a-8f2b-11e8-9355-63b608b88d56.png)
fillMat definition:
```js
const fillMat = new ColorMaterialProperty(new Color(1, 1, 1, 0.4));
```

**Comment by hpinkos** [comment]
Hello @golyalpha,  CesiumJS does not officially support TypeScript.  There has been some discussion here: #5717

We're always happy to take a pull request to make Cesium easier to use with TypeScript!  For example, see this PR with `tds-jsdoc` fixes: #6767

**Comment by mramato** [comment]
Just to clarify, we officially support using Cesium in TypeScript applications, we just do not have an official TypeScript definition file.  Lots of our users are using TS for development.

In your specific case, `outlineColor` can either be a `Color` or a `Property` that evaluates to a color.  It can not be a `ColorMaterialProperty` as `outlineColor` is not a `Material`.
